backend/activity_log/api_views.py

```diff
@@ -55,9 +55,25 @@ class EventTypeViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.EventTypeSerializer
     queryset = models.EventType.objects.order_by("name")
     filter_backends = [DjangoFilterBackend]
-
+    filterset_fields = ["timestamp"]
     permission_classes = [core_permissions.IsReadOnly & permissions.IsAuthenticated]
 
+    def get_queryset(self):
+        hour_ago = timezone.now()-timezone.timedelta(hours=1)
+        query = models.LogEntry.objects.annotate(
+            date=Cast("timestamp", output_field=DateField())
+        )
+        query = query.values("date").annotate(total=Count("date"))
+        query = query.order_by("-date")
+        query = query.filter(timestamp__gte=hour_ago,timestamp__lte=timezone.now())
```

Comment:
This function is very strange. Why are you only showing the stuff that happened in the last hour?  You are making sure most data is inaccessable.  This looks a lot like wild guessing.